src/Framework/MSBuildEventSource.cs

```diff
@@ -0,0 +1,465 @@
+﻿using System;
+using System.IO;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+
+
+//
+// This captures information of how various key methods of building with MSBuild ran.
+//
+namespace Microsoft.Build.Eventing
+{
+    // /OnlyProviders=*MSBuild-Profiling
```

Comment:
Is this comment useful? If it is, is it stale?

Reply:
Useful, yes, stale, also yes. Will fix.